December review (#162)
hassanobeid1994/tr_b_causal_2020: article/sections/_9_conclusion.tex

\section{Conclusion}
\label{sec:conclusion}

Travel demand problems aim at forecasting the impact of proposed project and policies. These problems are causal in nature. 

> To our surprise, there has been very little mention of causal inference in the field of travel demand modeling.

Author: Overall, given that this is a conclusion, can we be as broad as possible by relating our work not just to transportation and travel demand modeling but to choice modeling overall.

Also, I don't know if the slow adoption is surprising as much as sad. I'm suggesting removing very as well simply because its the literary equivalent of typing in all caps.
```suggestion
To date, choice modelers have not routinized the use of causal inference in the field of travel demand modeling.
```

\citet{brathwaite_2018_causal} have documented this disconnect and presented an initial framework for addressing it.

> In this chapter, we build up on \citet{brathwaite_2018_causal} and highlight the importance of using causal graphs and 
> causal inference methods in transportation demand modeling efforts.

Author: I suggest removing the citation as it seemed visually repetitive with the two citations only being separated by a sentence.

```suggestion
In this chapter, we built upon their framework.
We highlighted the importance of using causal graphs and
causal inference methods in transportation demand modeling efforts and in choice modeling more generally.
```

We presented a numerical exercise aiming to show the importance of  the data generating process in the estimation 

> of effects of interest resulting from external interventions. 

Author: I think the conclusion is one place we can afford to go with the less precise but more common "treatment" effect phrase, e.g.
```suggestion
of treatment effects from external interventions. 
```

We showed, using the same outcome model, that different assumptions about the data generating processes could 
lead to bias in estimates of the effect of interest.

Data generating processes can be illustrated effectively using DAGs as shown by \citet{pearl_1995_causal}.
We have shown that DAGs help researchers and practitioners represent their assumptions about the data generating process. 

We presented a process allowing researchers and practitioners to construct causal graphs based on their expert opinion of the problem at hand. 

> We also presented methods for testing the implications encoded in the the practitioners proposed causal graph. 

Author: ```suggestion
We also presented methods for testing the implications encoded in the practitioners proposed causal graph. 
```

These implications might be either observed, latent, or both.

Beyond testing the implications and assumptions encoded in a one's causal graph, we describe why causal discovery is important in discerning relationships between covariates one's data.

> We shortly presented several causal discovery algorithms used by

Author: Total nitpick for vanity because shortly just felt like it more strongly connoted height than briefly connoted time.
```suggestion
We briefly presented several causal discovery algorithms used by
```
